### ISSUE TITLE
[Bug][a11y] X to close tab is difficult to tap

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/fenix#4193]
## Steps to reproduce
- Open a tab
- Open tabs tray
- Click X on tab to close

### Expected behavior
Tab closes

### Actual behavior
Often, tab is selected instead.

Android accessibility guidelines suggest that minimum touch target size should be 48dp: https://developer.android.com/guide/topics/ui/accessibility/apps#touch-targets Visibly, the button doesn't look that large nor does the touch target feel that large.

The other way to close a single tab is to swipe it closed, however, afaik, that is also not an accessible gesture.

### Device information
* Android device: Pixel 2
* Fenix version: Nightly 190719

**Comment by DiegoARL38** [comment]
Most buttons in the Home screen are affected, actually.

**Comment by Dunexus** [comment]
Screenshot with layout bound :

![Screenshot_20190722-095141__01](https://user-images.githubusercontent.com/13906248/61615820-3efeb680-ac67-11e9-9569-45c5a4e5b445.jpg)

**Comment by sblatz** [comment]
Should be addressed by https://github.com/mozilla-mobile/fenix/pull/3801.

Can QA please verify?

**Comment by sv-ohorvath** [comment]
@sblatz The button size is now larger, but the touch target is the same 24dp, according to the A11y scanner. Is that enough to close this?

**Comment by sblatz** [comment]
No, let's keep this open until that is addressed :) Thanks!

**Comment by softvision-miralobontiu** [comment]
Verified as fixed on RC 2.0.0-rc.2 with Samsung Galaxy Tab S3 (Android 8), and OnePlus 5T (Android 9).
